entrypoint.sh

```diff
@@ -2,6 +2,10 @@
 
 IMAGE_NAME="$INPUT_GCR_HOST/$INPUT_GCR_PROJECT_ID/$INPUT_IMAGE_NAME:$INPUT_IMAGE_TAG"
 
+
 echo -n $GCLOUD_SERVICE_KEY | docker login -u _json_key --password-stdin https://$INPUT_GCR_HOST/
-docker build -t $IMAGE_NAME  .
+
+cd $INPUT_WORKING_DIR
```

Comment:
in projects that will have multiple folders with a docker file, it will be inconvinient, or event impossible to copy the binray to the image, since the docket build is not rooted in a directory that contains the binary.

i mean that you will need to do

COPY ../build/binary .


in a Dockerfile

not sure you are allowed to do .. ( go directory up, above the root of the docker build )
a better approach imo is to use the -f flag
```

docker build -h

  -f, --file string             Name of the Dockerfile (Default is 'PATH/Dockerfile')
```

the docker files can still be in a subdirectory. but the docker build will be rooted in the root of the repository.
which is better inmo.
or you can have the docker files flat in the root. also a good option.

Reply:
I had the problem your'e mentioning with the -f flag.
Thats why I used the cd approach.